tests/dim/byref-shared-from-shared-ref.bas

```diff
@@ -1,4 +1,4 @@
-' TEST_MODE : COMPILE_ONLY_FAIL
+' TEST_MODE : COMPILE_ONLY_OK
```

Comment:
we can probably remove this now, since the scenario is better covered by the new unit test

Reply:
done